lib/widgets/ln_fee_message.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:math';
+
+import 'package:breez_sdk/bridge_generated.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/account/account_bloc.dart';
+import 'package:c_breez/bloc/account/account_state.dart';
+import 'package:c_breez/bloc/currency/currency_bloc.dart';
+import 'package:c_breez/bloc/lsp/lsp_bloc.dart';
+import 'package:c_breez/bloc/payment_options/payment_options_bloc.dart';
+import 'package:c_breez/widgets/warning_box.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LnAddressFeeMessage extends StatelessWidget {
+  const LnAddressFeeMessage();
+
+  @override
+  Widget build(BuildContext context) {
+    final themeData = Theme.of(context);
+    final lspState = context.watch<LSPBloc>().state;
+    final isChannelOpeningAvailable = lspState?.isChannelOpeningAvailable ?? false;
+    final openingFeeParams = lspState?.lspInfo?.openingFeeParamsList.values.first;
+
+    return isChannelOpeningAvailable
+        ? WarningBox(
+            boxPadding: const EdgeInsets.fromLTRB(16, 30, 16, 16),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  _formatFeeMessage(context, openingFeeParams!),
+                  style: themeData.textTheme.titleLarge,
+                  textAlign: TextAlign.center,
+                ),
+              ],
+            ),
+          )
+        : const SizedBox();
+  }
+
+  String _formatFeeMessage(BuildContext context, OpeningFeeParams openingFeeParams) {
+    final texts = context.texts();
+    final currencyState = context.read<CurrencyBloc>().state;
+    final accountState = context.read<AccountBloc>().state;
+    final paymentOptionsState = context.read<PaymentOptionsBloc>().state;
+
+    final minFee = openingFeeParams.minMsat ~/ 1000;
+    final minFeeFormatted = currencyState.bitcoinCurrency.format(minFee);
+    final minFeeAboveZero = minFee > 0;
+    final setUpFee = (openingFeeParams.proportional / 10000).toString();
+    final liquidity = currencyState.bitcoinCurrency.format(
+      accountState.maxInboundLiquidity,
+    );
+    final liquidityAboveZero = accountState.maxInboundLiquidity > 0;
+
+    final channelFeeLimitSat = paymentOptionsState.channelFeeLimitMsat ~/ 1000;
+    final maxFee = maxReceivableSat(accountState, channelFeeLimitSat, openingFeeParams);
+    final maxFeeFormatted = currencyState.bitcoinCurrency.format(maxFee);
+    final isFeesApplicable = maxFee > accountState.maxInboundLiquidity;
+
+    if (!isFeesApplicable) {
+      // Send more than {minSats} and up to {maxSats} to this address. This address can be used only once.
+      return texts.invoice_btc_address_channel_not_needed(minFeeFormatted, maxFeeFormatted);
```

Comment:
These texts are confusing me. LnAddressFeeMessage seems related to the LnAddressWidget but the texts are referencing btc addresses.

Reply:
We are reusing the same texts between btc address & ln address fee information as we currently don't have any texts specific for ln adresses. This feature was not previously available on Breezmobile. 

I'll duplicate them for `ln_address` on [Breez-Translations](https://github.com/breez/Breez-Translations) just in case we need to change the wording, handle a new case & require a new text etc. so it's less likely to have any regressions for btc address texts.

Reply:
Added missing messages & renamed existing messages to properly reflect where they are used.

- [Update Breez-Translations](https://github.com/breez/c-breez/pull/842/commits/2f8225bc577a8dacdeaa7e8b42ece44c83c4b374)